src/components/Campaign.js

```diff
@@ -1,48 +1,55 @@
 import React from "react";
-
-import CampaignTile from "./CampaignTile";
+import Card from "./Card";
 
 const Campaign = () => (
 	<div className="container">
 		<div className="row">
-			<div className="col-xl-2 col-md-3">
-				<CampaignTile
-					href="https://jobactive.gov.au/path"
-					imgSrc="https://www.australia.gov.au/sites/default/files/styles/homepage_quick_link_carousel/public/global_site/library/images/carousel_images/ausgovau_YouthJobsPaTH_259x153.jpg?itok=goqS2o8J"
+			<div className="col-md-3">
```

Comment:
I think we should have the `col-xl-2` tag in all divs here

Reply:
So it looks like this?

![image](https://user-images.githubusercontent.com/20184809/44242775-e7fa9100-a20e-11e8-8169-26a5817d0ead.png)